packages/application-shell/src/components/quick-access/create-commands.ts

```diff
@@ -57,7 +57,7 @@ const createCommands = ({
         keywords: ['Go to Products'],
         action: {
           type: actionTypes.go,
-          to: `/${applicationContext.project.key}/products`,
+          to: `/${applicationContext.project.key}/products/pim-search`,
```

Comment:
Shouldn't the products app have a redirect `products` -> `products/pim-search`? 🤔

Reply:
Once the old list is removed yes. That was always my understanding. I always thought we have to alterantives when the old list is removed

1. Redirect from `products` to `products/pim-search`
2. Make `products/pim-search` the default instead of `products`

I somewhat like 2. as the routing is cleaner which would mean we wouldn't change anything here. If we do we'd have to change it back again.

Reply:
Yes, and we will do that, except not now because we still didn't remove the old product list from the application, once we do that, we will rewrite the URLs and add the redirect. Meanwhile, we're doing a quick fix here that won't need to be changed later thanks to the redirect.

Reply:
Ok

Reply:
> I somewhat like 2. as the routing is cleaner which would mean we wouldn't change anything here. If we do we'd have to change it back again.

We will go with 1. because `products/pim-search` was meant to be a temporary route until the PIM list feature is released, and `products` is more clean and straightforward name. In addition, we will create a redirect from `products/pim-search` to `products` (once we bind the PIM list to `products`) so that users used to `products/pim-search` won't be lost. And thanks to this redirect we won't need to change the fix implemented in this PR.

Reply:
Ah, I think you mean by 1. what I meant by 2. I prefer the way you describe. So numbers don't matter :)